tests/integration/routes/account-test.js

```diff
@@ -209,11 +209,86 @@ getServer(function (error, server) {
     group.end()
   })
 
-  test('PATCH /session/account', {todo: true}, function (t) {
-    // couchdbErrorTests(server, group, couchdbPutUserMock, patchAccountRouteOptions)
-    // invalidTypeErrors(server, group, patchAccountRouteOptions)
+  test('PATCH /account/abc4567', function (group) {
+    var patchAccountRouteOptionsPassword = {
+      method: 'PATCH',
+      url: '/account/abc4567',
```

Comment:
the path is `/session/account`, it’s from the perspective of the currently authenticated user, so there is no id in the path